lib/associations/associations.rb

```diff
@@ -4,16 +4,16 @@ module Associations
     module ActiveRecordExtensions
 
       def belongs_to(name, scope = nil, **options)
-        options = {:class_name => name.to_s.camelize }.merge(options)
+        klass_name = options.key?(:class_name) ? options[:class_name] : name.to_s.camelize
         klass =
           begin
-            options[:class_name].constantize
-          rescue
-            nil
-          rescue LoadError
+            klass_name.constantize
+          rescue StandardError, LoadError
```

Comment:
@koyo-miyamura Would you document why you need to add a `StandardError` here?

Reply:
@kbrock 
I'm sorry but I cannot document this, because I only refactored this code.

Original code also handle `StandardError` but I don't explain why.